classes/class-wc-connect-shipping-label.php

```diff
@@ -39,7 +39,7 @@ protected function get_items_as_individual_packages( $order ) {
 				$name = html_entity_decode( $product->get_formatted_name() );
 
 				for ( $i = 0; $i < $item[ 'qty' ]; $i++ ) {
-					$packages[] = array(
+					$packages[ "weight_{$i}_individual" ] = array(
```

Comment:
The `{}` are not needed. And if you put them (for clarity), the correct way is `"weight_${i}_individual"`